### ISSUE TITLE
FusionCMS not working on Ubuntu 18.04

### ISSUE THREAD

**Issue by Nocty1501** [Yekta-Core/FusionCMS#1]
After completing the installation process and deleting the "install" folder, the website will 404 on each page, except for /index.php/login, which will give a PHP error.

**Comment by BarbzYHOOL** [comment]
You don't have the mod_rewrite working

**Comment by TrekPL** [comment]
> 
> 
> After completing the installation process and deleting the "install" folder, the website will 404 on each page, except for /index.php/login, which will give a PHP error.

Exactly the same problem, and YES - I do have mod_rewrite enabled, as well as all required modules in apache and extensions in PHP.
There are no errors in apache log, but in CMS log I can see: `ERROR - 2018-10-17 12:06:17 --> 404 Page Not Found: /index` - wrong file name somewhere? Without .php at the end?

**Comment by BarbzYHOOL** [comment]
then check the .htaccess (if it's even read)

**Comment by TrekPL** [comment]
> 
> 
> then check the .htaccess (if it's even read)

Yes, apache got full access to it. Main htaccess is exactly the same as used in other copies of FCMS that I've tried and on those it was working. So I presume installer is missing some step(s?),
Take a note that I'm using PHP7.2 and Apache/2.4.29. I made a fresh install and found those in apache error log:
`[Wed Oct 17 17:57:25.495548 2018] PHP Warning:  fwrite() expects parameter 1 to be resource, boolean given in /var/www/XXXXX/install/system.php on line 49, referer: http://XXXXX/install/
[Wed Oct 17 17:57:25.495779 2018] [php7:warn] [pid 7473] [client XXXXX:57111] PHP Warning:  fclose() expects parameter 1 to be resource, boolean given in /var/www/XXXXX/install/system.php on line 50, referer: http://XXXXX/install/
[Wed Oct 17 17:57:25.495860 2018] [php7:warn] [pid 7473] [client XXXXX:57111] PHP Warning:  unlink(../application/cache/write_test.txt): No such file or directory in /var/www/XXXXX/install/system.php on line 52, referer: http://XXXXX/install/`
No other errors in log.
I even tried to add realm data to DB as installer do not pass them into DB. Also no effect. I will try few more things today and give a feedback.

**Comment by BarbzYHOOL** [comment]
try with an older fcms and if it doesn't work then it means this is this version that is bugged (i use an older fcms)

**Comment by yuanf225** [comment]
Yes, I have the same problem

**Comment by Nightprince** [comment]
i know :| i fix it , but i don't want to pull this.
Because some people published this in their own name

**Comment by Nightprince** [comment]
> Yes, I have the same problem



> try with an older fcms and if it doesn't work then it means this is this version that is bugged (i use an older fcms)

i know :| i fix it , but i don't want to pull this.
Because some people published this in their own name

**Comment by jdhirst** [comment]
@Nightprince Would you be willing to provide me the fix? I would very much appreciate it as its for my own site, not to be published as a separate project.
